service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/ReviewCMOServiceTest.java

```diff
@@ -0,0 +1,227 @@
+import static uk.gov.hmcts.reform.fpl.enums.HearingType.CASE_MANAGEMENT;
+import static uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicListElement.EMPTY;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, FixedTimeConfiguration.class, ReviewCMOService.class})
+class ReviewCMOServiceTest {
+
+    private static final String SINGLE = "SINGLE";
+    private static final String MULTI = "MULTI";
+    private static final String NONE = "NONE";
+    private static final String hearing1 = "Case management hearing, 2 March 2020";
+    private static final String hearing2 = "Test hearing, 15 October 2020";
+    private static final DocumentReference order = testDocumentReference();
+
+    @Autowired
+    private ReviewCMOService service;
+
+    @Autowired
+    private Time time;
+
+    @Test
+    void shouldReturnMultiPageDataWhenThereAreMultipleDraftCMOsReadyForApproval() {
+        List<Element<CaseManagementOrder>> draftCMOs = List.of(draftCMO(hearing1), draftCMO(hearing2));
+        CaseData caseData = CaseData.builder().draftUploadedCMOs(draftCMOs).build();
+
+        Map<String, Object> expectedData = Map.of(
+            "numDraftCMOs", MULTI,
+            "cmoToReviewList", DynamicList.builder()
+                .value(EMPTY)
+                .listItems(dynamicListItems(draftCMOs.get(0).getId(), draftCMOs.get(1).getId())).build());
+
+        assertThat(service.handlePageDisplayLogic(caseData)).isEqualTo(expectedData);
+    }
+
+    @Test
+    void shouldReturnSinglePageDataWhenThereIsOneDraftCMOReadyForApproval() {
+        CaseData caseData = CaseData.builder().draftUploadedCMOs(List.of(draftCMO(hearing1))).build();
+
+        Map<String, Object> expectedData = Map.of(
+            "numDraftCMOs", SINGLE,
+            "reviewCMODecision", ReviewDecision.builder()
+                .hearing(hearing1)
+                .document(order).build());
+
+        assertThat(service.handlePageDisplayLogic(caseData)).isEqualTo(expectedData);
+    }
+
+    @Test
+    void shouldReturnNonePageDataWhenThereAreNoDraftCMOsReadyForApproval() {
+        CaseData caseData = CaseData.builder().draftUploadedCMOs(List.of()).build();
+
+        Map<String, Object> expectedData = Map.of(
+            "numDraftCMOs", NONE);
+
+        assertThat(service.handlePageDisplayLogic(caseData)).isEqualTo(expectedData);
+    }
+
+    @Test
+    void shouldReturnCMOsThatAreReadyForApproval() {
+        Element<CaseManagementOrder> draftCMO1 = draftCMO(hearing1);
+        Element<CaseManagementOrder> draftCMO2 = draftCMO(hearing2);
+        Element<CaseManagementOrder> draftCMO3 = element(CaseManagementOrder.builder().status(RETURNED).build());
+        Element<CaseManagementOrder> draftCMO4 = element(CaseManagementOrder.builder().status(APPROVED).build());
```

Comment:
it is hard to understand why first 2 are so special that they should be returned as ready for approval.
If you do:
        Element<CaseManagementOrder> draftCMO1 = element(CaseManagementOrder.builder().status(SEND_TO_JUDGE).build());
        Element<CaseManagementOrder> draftCMO2 = element(CaseManagementOrder.builder().status(SEND_TO_JUDGE).build());

then you immediately see that status is a factor that make them special

unit tests are not only to protect from regression, it is great way of learning how system works, but must be easy to read and understand